copier/template.py

```diff
@@ -435,6 +435,15 @@ def templates_suffix(self) -> str:
             return DEFAULT_TEMPLATES_SUFFIX
         return result
 
+    @cached_property
+    def preserve_symlinks(self) -> bool:
+        """Get whether to preserve symlinks
+
+        By default: `True`
+
+        See [preserve_symlinks][]"""
+        return bool(self.config_data.get("preserve_symlinks", True))
```

Comment:
> I'd just add an option to copier.yaml: `_symlinks` with choices `as_symlinks` or `as_files`.

Sorry, I didn't see this code. Your option name seems better to me. And I like it more that it is a boolean. Please leave it as you've done, thanks!